dist/icon/ds6/icon.css

```diff
@@ -17,3 +17,15 @@ svg.icon--hamburger {
   height: 14px;
   width: 12px;
 }
+svg.icon--close {
+  height: 17px;
+  width: 17px;
+}
+span.icon--close {
+  display: inline-block;
```

Comment:
Feels like maybe the `display`, `vertical-align` and `background-repeat` properties could form the `span.icon` base class for all icons.